go/cmd/goserver/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"github.com/feast-dev/feast/go/internal/config"
+	"github.com/feast-dev/feast/go/internal/feast"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"os"
+)
+
+const (
+	flagFeastRepoPath    = "FEAST_REPO_PATH"
+	flagFeastRepoConfig  = "FEAST_REPO_CONFIG"
+	flagFeastGrpcPort    = "FEAST_GRPC_PORT"
+	defaultFeastGrpcPort = "6566"
+	feastServerVersion   = "0.18.0"
+	defaultFeastHttpPort = "8081"
+)
+
+// TODO: Add a proper logging library such as https://github.com/Sirupsen/logrus
+
+func main() {
+
+	repoPath := os.Getenv(flagFeastRepoPath)
```

Comment:
There's very neat lib to do this w/o manual parsing for each variable
https://github.com/kelseyhightower/envconfig